runtime/account_test.go

```diff
@@ -2676,6 +2676,49 @@ func TestRuntimeGetAuthAccount(t *testing.T) {
 
 		assert.IsType(t, &sema.NotDeclaredError{}, errs[0])
 	})
+
+	t.Run("update names index expression start position", func(t *testing.T) {
```

Comment:
Nice work adding this test!

Tests in the `runtime` package are usually "integration tests" in a way. As in, it setup the environments, create transactions, etc. For this change, given we only need to test the interpreting behaviour, moving this test to the interpreter unit tests under `runtime/tests/interpreter` (`indexing_test.go` file) might be a better option.

Reply:
Given there are multiple equivalent tests in `runtime/tests/interpreter/interpreter_test.go` which cover the case of errors being thrown using the update index expression positions, this test has been removed.

Reply:
make sense 👍